pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1501,4 +1502,28 @@ self: super: {
   # Fixed at head, but hasn't cut a release in awhile.
   darcs = doJailbreak super.darcs;
 
+  # 20.03 broken packages
+
+  envy-extensible = markBroken super.envy-extensible;
+  exceptionfree-readfile = markBroken super.exceptionfree-readfile;
+  fcf-containers = markBroken super.fcf-containers;
+  first-class-instances = markBroken super.first-class-instances;
+  flp = markBroken super.flp;
+  hal = markBroken super.hal;
+  hcad = markBroken super.hcad;
+  hedis-envy = markBroken super.hedis-envy;
+  hhwloc = markBroken super.hhwloc;
+  layered-state = markBroken super.layered-state;
+  network-uri-static = markBroken super.network-uri-static;
+  pandoc-plot = markBroken super.pandoc-plot;
+  patch = markBroken super.patch;
+  prologue = markBroken super.prologue;
+  provenience = markBroken super.provenience;
+  refractor = markBroken super.refractor;
+  shwifty = markBroken super.shwifty;
+  smallcheck-kind-generics = markBroken super.smallcheck-kind-generics;
+  traversal-template = markBroken super.traversal-template;
+  vector-text = markBroken super.vector-text;
+  zbar = markBroken super.zbar;
+
```

Comment:
@cdepillabout What about stable? I had @disassembler do this because I knew on master these are semi-automatically marked broken, but we both had no idea about stable.

Reply:
`release-20.03` has greatly diverged from `master`. We're based on a very different package set and a different compiler, too. Porting those "mark broken" overrides into `master` makes little sense. Generally speaking, we mark broken packages as such semi-automatically once per week anyway. Please don't merge those changes into master.

Reply:
@peti I understand because of the divergence.